app/assets/stylesheets/components/_filter-panel.scss

```diff
@@ -14,44 +14,40 @@
   align-items: center;
   gap: govuk-spacing(2);
 
-  .gem-c-heading {
-    font-weight: normal;
+  .app-c-filter-panel__count {
```

Comment:
Does this need to be nested beneath `.app-c-filter-panel__content`? We try to avoid nesting if possible, particularly since we're using BEM here.